src/neptune/internal/utils/uncaught_exception_handler.py

```diff
@@ -76,4 +74,10 @@ def unregister(self, uid: uuid.UUID):
                 del self._handlers[uid]
 
 
+def exception_handler(uncaught_exception_handler: UncaughtExceptionHandler, exc_type, exc_val, exc_tb):
+    uncaught_exception_handler.trigger(exc_type, exc_val, exc_tb)
+
+    uncaught_exception_handler._previous_uncaught_exception_handler(exc_type, exc_val, exc_tb)
```

Comment:
Let's add a validation that it isn't `None`; I think that this would be detected by `mypy`, don't you wanna add proper typings? Doesn't look to be demanding :wink: